EasyRestProjectNetTeam2/EasyRestPages/SignUpPage.cs

```diff
@@ -28,33 +30,34 @@ public SignUpPage(IWebDriver driver) : base(driver)
         [FindsBy(How = How.XPath, Using = "//span[text()='Create account']")]
         private IWebElement _createAccountButton;
 
+        [FindsBy(How = How.XPath, Using = "//input[@name='birthDate']")]
+        private IWebElement _inputBirthDate;
 
 
         public void SendKeysToInputName(string name)
         {
             _inputName.SendKeys(name);
         }
-
+        public void ClickToInputBirthDate()
+        {
+            _inputBirthDate.Click();
```

Comment:
It is better to return the DatePickerComponent instance here, after the Click() method, and in the constructor, because that component can appear only when the user initiates this.